### DISCUSSION TITLE
Release v1.9.3 / 5.64.3

### DISCUSSION THREAD

**Discussion by DavidXanatos** [sandboxie-plus/Sandboxie#2886] [Announcements]
![0 5 5](https://user-images.githubusercontent.com/3890945/105014988-575cb680-5a41-11eb-90d5-d942d28184b2.png)

This build fixes a few issues with 1.9.2

Please review the full [Changelog](https://github.com/sandboxie-plus/Sandboxie/blob/master/CHANGELOG.md#193--5643---2023-05-08) for more details.

#### You can support the project through donations, any help will be greatly appreciated.

[<img src="https://www.paypalobjects.com/en_US/AT/i/btn/btn_donateCC_LG.gif">](https://xanasoft.com/go.php?to=donate)

#### If you have issues with an update installation, just uninstall the previous version keeping the sandboxie.ini and reinstall the new build.

<hr /><em>This discussion was created from the release <a href='https://github.com/sandboxie-plus/Sandboxie/releases/tag/v1.9.3'>Release v1.9.3 / 5.64.3</a>.</em>

**Comment by BilleBarrett** [comment]
Since I upgrade I'm getting these errors many times
robotaskbaricon.exe (14652): SBIE2203 Failed to communicate with Sandboxie Service:  *GUIPROXY_00000001; MsgId: 19 - robotaskbaricon.exe [00000102]
11:57:40.117   chrome.exe (26428): SBIE2321 Cannot manage device map:  [C0000034 / 11] 
Is there a setting I'm missing
I also found that I had to run Zoom outside of Sadnboxie for a client
